### DISCUSSION TITLE
More than one Pylontech battery supported with OpenDTU-OnBattery?

### DISCUSSION THREAD

**Discussion by lexxman123** [hoylabs/OpenDTU-OnBattery#543] [General]
Hi all,

is it possible to use more than one Pylontech battery (connected to each other via the Link port) with OpenDTU-OnBattery connected to the first / master battery (as shown in the picture)?
Or is it not supported with the current software implementation? 

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/77163863/7370f8ca-99bc-4c1e-a6bf-567019b1ebb6)

**Comment by JaMo523** [comment]
Is supported. I have 2 stacked Pylontech.
